processing/src/main/java/org/apache/carbondata/processing/datatypes/PrimitiveDataType.java

```diff
@@ -330,21 +331,35 @@ public void writeByteArray(Object input, DataOutputStream dataOutputStream,
     }
   }
 
+  private byte[] getNullForBytes(byte[] value) {
```

Comment:
i think this is not required, can reuse `updateNullValue`, please check

Reply:
removed.